src/calls.c

```diff
@@ -2006,17 +1993,17 @@ Obj FuncHandlerCookieOfFunction(Obj self, Obj func)
 */
 void SaveFunction ( Obj func )
 {
-  UInt i;
-  for (i = 0; i <= 7; i++)
-    SaveHandler(HDLR_FUNC(func,i));
-  SaveSubObj(NAME_FUNC(func));
```

Comment:
Here you seem to switch from `NAME_FUNC` to `header->name`, but in other places you continue using the functions. Are you planning on using the functions long term?
Also, I would guess `LoadSubObj` (and maybe `SaveSubObj`?) could cause GC to occur, so is it safe to keep `header` throughout `SaveFunction` and `LoadFunction`?